spec/system/provider_interface/provider_manages_application_interviews_spec.rb

```diff
@@ -52,6 +52,42 @@
     when_i_cancel_an_interview
     i_can_see_the_application_is_awaiting_provider_decision
     and_the_interview_tab_is_not_available
+
+    # TODO: the "make decision" prompt does not show here because of the flash message
```

Comment:
Can we add the card number to this TODO (and mention the existence of the TODO on the relevant card?)